tests/cli/test_rasa_data.py

```diff
@@ -91,19 +91,6 @@ def test_data_split_help(run: Callable[..., RunResult]):
         assert line in printed_help
 
 
-def test_data_convert_help(run: Callable[..., RunResult]):
```

Comment:
I actually think this test is still useful and and should only amended by removing the `md` from `help_text`?